docs/USAGE.md

```diff
@@ -59,6 +59,14 @@ To disable or enable dynamic fallback use the the following API (When enabled, c
     openvino_tensorflow.enable_dynamic_fallback()
     openvino_tensorflow.disable_dynamic_fallback()
 
+To export the translated IRs of the clusters use the API below. This will dump the clusters from the most recently executed model as ".xml" and ".bin" files which can be used for an OpenVINO application later. The first parameter to this API is the output directory. If there is any pre-existing cluster file in the corresponding library, it will ask user to confirm before overwriting any of the older files. To disable this check, pass a "False" value as the second parameter(optional). Then, any pre-existing IR file will be overwritten without any confirmation if the cluster name is same.
```

Comment:
Should it be "corresponding directory" instead of "corresponding library"?

Reply:
Yes, that was a mistake. Fixed it.